shared/templates/docker/setup_data_libraries.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+import os
```

Comment:
This script should be in https://github.com/galaxyproject/ephemeris and then be part of the main image.
Please ping me and hit me with a stick if I forget to do this.